templates/crossplane/apis/types.go.tpl

```diff
@@ -5,18 +5,15 @@
 package {{ .APIVersion }}
 
 import (
-	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
-	"github.com/aws/aws-sdk-go/aws"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // Hack to avoid import errors during build...
 var (
 	_ = &metav1.Time{}
-	_ = &aws.JSONValue{}
```

Comment:
You won't need the ackv1alpha1 import, yes. But just double-check you won't need this one above...